includes/Modules/Idea_Hub.php

```diff
@@ -121,7 +121,7 @@ function( $maybe_empty, $postarr ) {
 		$this->post_name_setting = new Post_Idea_Name( $post_meta );
 		$this->post_name_setting->register();
 
-		$this->post_text_setting = new Post_Idea_Text( $post_meta );
+		$this->post_text_setting = new Post_Idea_Text( $post_meta, array( 'show_in_rest' => true ) );
```

Comment:
This is what I meant by moving the source of truth. A simple `bool` for `show_in_rest` isn't a good idea either, because complex values (like JSON arrays, etc.) require a transform to handle how they should be displayed. We could add that as some extra `Post_Meta` metadata, I suppose.